resources/views/partials/item/title.blade.php

```diff
@@ -7,7 +7,7 @@
 }">
     <div class="flex items-center" slot-scope="{ toggle, close, isOpen }">
         <template v-if="!isOpen">
-            <span class="text-base text-primary truncate peer place-self-start -mt-0.5">
+            <span class="text-primary truncate peer place-self-start -mt-0.5">
```

Comment:
peer on span?

Reply:
Yes, because when we hover over the span an icon will show so you can edit the name of the wishlist as you can see below. 

```
<x-heroicon-s-pencil-square
    class="pl-2 w-6 h-4 cursor-pointer shrink-0 opacity-0 hover:opacity-100 peer-hover:opacity-50 transition-opacity"
    v-on:click="toggle"
/>
```